app/src/main/java/bg/zahov/app/data/provider/WorkoutProviderImpl.kt

```diff
@@ -111,6 +112,14 @@ class WorkoutProviderImpl : WorkoutProvider {
         workoutRepo.updateTemplateWorkout(workoutId, date, newExercises)
     }
 
+    override suspend fun getPastWorkoutsForCurrentMonthWithWorkoutCount(): Flow<Pair<List<Workout>, Int>> {
```

Comment:
No need for such a long name. Most of this information and more if needed can be added to the kdoc.

Reply:
👍